src/test/java/org/mockito/internal/stubbing/answers/ThrowsExceptionTest.java

```diff
@@ -28,6 +28,62 @@ public void should_raise_wanted_throwable() throws Throwable {
         }
     }
 
+    /**
+     * Tests that ThrowsException raises the same instance as passed in the constructor
+     * in case of a well-behaved Throwable implementation.
+     *
+     * @see #should_raise_same_wanted_throwable_even_if_it_returns_null_in_fill_in_stack_trace()
+     */
+    @Test
+    public void should_raise_same_wanted_throwable_on_subsequent_invocations() {
```

Comment:
I like this test. Can you check if we have top level integration tests for those use cases? I did a quick search and I think we have some gaps.

I'm wondering if top level tests under 'org.mockitousage.*' packages aren't better to verify and document this behavior. ThrowsExceptionTest is becoming big and complicated and it does seem to repeat the coverage of the other tests. Thoughts?

Reply:
There is a similar integration test: `org.mockitousage.stubbing.StubbingConsecutiveAnswersTest#should_throw_consecutively`

Shall I remove this one?